lib/tests/test_fun_call.ml

```diff
@@ -150,13 +150,13 @@ let%expect_test "wrap_callback_strict" =
     Result: function#0 |}];
   call_and_log
     (Js.Unsafe.callback_with_arity 2 cb3)
-    {| (function(f){ return f(1,2,3)(4) }) |};
+    {| (function(f){ return caml_callback(f(1,2,3),[4]) }) |};
```

Comment:
These tests don't work with `dune runtest --profile release` because short-var is enabled and `caml_callback` is renamed.